configs/sandpack-contents/component-theming/code.js

```diff
@@ -0,0 +1,109 @@
+module.exports = {
+  App: `import { Box, SimpleGrid, IconButton, Code, useColorMode } from "@chakra-ui/react";
+import { FaMoon, FaSun } from "react-icons/fa";
+
+export default function App() {
+  const { toggleColorMode, colorMode } = useColorMode();
+  return (
+    <Box position="relative" h="100vh">
+      <SimpleGrid gap={12} p={12} columns={2}>
+        <Code>
+          Themed solid code
+        </Code>
+        <Code variant="custom">
+          Themed custom code
+        </Code>
+        <Code variant="solid">
+          Themed solid code
+        </Code>
+        <Code variant="outline">
+          Themed outline code
+        </Code>
```

Comment:
Can you redesign these? It's not a suitable look for the `Code` component.

Reply:
sure i'll add something else